test/parallel/test-runner-exit-code.js

```diff
@@ -16,11 +16,11 @@ async function runAndKill(file) {
   });
   const [code, signal] = await once(child, 'exit');
   await finished(child.stdout);
-  assert.strictEqual(code, 1);
-  assert.strictEqual(signal, null);
   if (common.isWindows) {
     common.printSkipMessage('signals are not supported in windows');
   } else {
+    assert.strictEqual(signal, null);
+    assert.strictEqual(code, 1);
     assert.match(stdout, /not ok 1/);
     assert.match(stdout, /# cancelled 1\n/);
```

Comment:
Can we move the stdout checks above the signal and code ones? It makes the error message much more useful when the test fails – as is, the message would be `expected 0 to equal to 1`, with my proposal it would show the test output.